src/main/java/capstone/mfslbackend/model/FantasyWeek.java

```diff
@@ -32,7 +33,18 @@ public class FantasyWeek {
     @JsonIgnore
     @ManyToOne(optional = false)
     @JoinColumn(name = "fantasy_team_id", nullable = false)
```

Comment:
change the name to fantasy_team_a_id to fix the checkstyle issue i think

Reply:
> change the name to fantasy_team_a_id to fix the checkstyle issue i think

it didn't fix it